services/api/blocksim_ratelimiter.go

```diff
@@ -112,7 +117,8 @@ func (b *BlockSimulationRateLimiter) Send(
 	}
 
 	// Create and fire off JSON-RPC request
-	if payload.Version == spec.DataVersionElectra {
+	// TODO - bharath: Does the electra flashbots_validateBuilderSubmissionV4 work for fulu?
+	if payload.Version == spec.DataVersionElectra || payload.Version == spec.DataVersionFulu {
```

Comment:
We need to wait for the `flashbots_validateBuilderSubmissionV5` impl for Fusaka being worked on here: https://github.com/paradigmxyz/reth/pull/17179

Reply:
this has been merged to main in reth. Once its rebased to the fusaka-devnet-2 branch, we should be able to use it in rbuilder.